### DISCUSSION TITLE
Is there a way to format the drive to ext4 using android?

### DISCUSSION THREAD

**Discussion by chorong761** [master-hax/pixel-backup-gang#18] [General]
Is there a way to format the drive to ext4 using android?

**Comment by master-hax** [comment]
i don't see gparted in [termux-packages](https://github.com/termux/termux-packages), so you'll likely have to compile  [gparted](https://gitlab.gnome.org/GNOME/gparted) for your device and use that.

**Comment by master-hax** [comment]
https://github.com/master-hax/pixel-backup-gang/issues/11#issuecomment-2320130871

@shadww says `mkfs` works in termux

**Comment by master-hax** [comment]
i'll mark this is closed since we found a solution - feel free to reopen if it doesn't work
